src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -47,13 +47,7 @@ <h5>{{ version }}</h5>
              id="clusterStatus">
       <div class="d-flex flex-column justify-content-center align-items-center">
         <ng-template #healthChecks>
-          <ul>
-            <li *ngFor="let check of healthData.health.checks">
-              <span [ngStyle]="check.severity | healthColor"
-                    [class.health-warn-description]="check.severity === 'HEALTH_WARN'">
-              {{ check.type }}</span>: {{ check.summary.message }}
-            </li>
-          </ul>
+          <cd-health-checks [healthData]="healthData"></cd-health-checks>
```

Comment:
Why not get pool health data as part of the component itself?

Reply:
I think its best to pass it over rather than doing another call in the component. This component looks like part of the health checks and whenever we invoke this I assume we will already have the healthData subscribed on the parent component (to populate other details like health status.. so rather than doing a subscription again in the child, its good to pass it.